easybuild/easyconfigs/b/berkeley_upc/berkeley_upc_translator-2.20.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+easyblock = 'ConfigureMake'
+
+name = 'berkeley_upc_translator'
+version = '2.20.2'
+
+homepage = 'http://upc.lbl.gov/'
+description = """Unified Parallel C (UPC) is an extension of the C programming
```

Comment:
this description doesn't cover this well, since it's the same as in the other easyconfig?